cmd/dp-code-list-api/main.go

```diff
@@ -30,14 +29,14 @@ func main() {
 		log.Error(err, nil)
 		os.Exit(1)
 	}
-	mongoDatastore, err := store.CreateMongoDataStore(cfg.MongoConfig)
+	neoDatastore, err := store.CreateNeoDataStore(cfg.Neo4jDatabaseAddress, cfg.Neo4jCodeListLabel, cfg.Neo4jPoolSize)
```

Comment:
If we just called this var 'datastore' or 'store' we wouldn't have to change it in the other places if we ever changed it again. But we may never change it again so probably doesn't matter.